api/kogito-events-core/src/main/java/org/kie/kogito/event/cloudevents/utils/CloudEventUtils.java

```diff
@@ -268,6 +268,20 @@ public static boolean safeBoolean(Boolean bool) {
         return bool != null && bool.booleanValue();
     }
 
+    public static void validateCloudEvent(Map<String, Object> cloudEvent) throws InvalidCloudEventException {
+        SpecVersion specVersion = SpecVersion.parse(String.valueOf(cloudEvent.get(CloudEventV1.SPECVERSION)));
+        switch (specVersion) {
+            case V1:
+                CloudEventValidatorV1.getInstance().validateCloudEvent(cloudEvent);
+                break;
+            case V03:
+                CloudEventValidatorV03.getInstance().validateCloudEvent(cloudEvent);
+                break;
+            default:
+                throw new UnsupportedOperationException("Unsupported CloudEvents version: " + specVersion);
+        }
```

Comment:
SpecVersion.parser already throws newInvalidSpecVersion, therefore this can be simplified
```suggestion
        SpecVersion specVersion = SpecVersion.parse(String.valueOf(cloudEvent.get(CloudEventV1.SPECVERSION)));
        BaseCloudEventValidator validator = specVersion == SpecVersion.V1 ? CloudEventValidatorV1.getInstance() : CloudEventValidatorV03.getInstance();
        return validator.validateCloudEvent(cloudEvent);
    }
```

Reply:
Done.